test/models/page_test.rb

```diff
@@ -13,4 +13,24 @@ class PageTest < ActiveSupport::TestCase
     page = Page.create(url: 'some/path/to/a/page')
     assert_not(page.valid?, 'The page should be invalid because it has no domain')
   end
+
+  test 'page title should sync with title from version with most recent capture time' do
+    page = pages(:home_page)
+    assert_equal('Page One', page.title)
+
+    page.versions.create(title: 'Newest Version', capture_time: '2017-03-05T00:00:00Z')
+    assert_equal('Newest Version', page.title, 'The page title should always sync against the title of the version with the most recent capture time')
```

Comment:
Alongside this assertion, you also need to make sure the changes were persisted to the DB (see note in the model file). You should get a fresh copy of the page from the DB to make assertions on:

```rb
page.versions.create(title: 'Newest Version', capture_time: '2017-03-05T00:00:00Z')

# Get a fresh page instance
page = Page.find(page.uuid) # OR: page.reload
assert_equal('Newest Version', page.title, '...')
# etc.
```

You could also check that the page wasn’t left with unsaved changes:

```rb
assert_not(page.changed?, 'The page was left with unsaved changes')
```